app.py

```diff
@@ -36,6 +36,30 @@ def list_bibliography_from_db(db, count):
     ).limit(count)
     return bibliography_sets.all()
 
+
+@app.route("/document_matrix")
```

Comment:
I'd like to stick to 1 word names for resources, let's call this just `matrix`, as we do in the CLI tool.